TypeCobol/TypeCobolResource.resx

```diff
@@ -166,4 +166,19 @@
   <data name="TypedefDeclaredOutProgramOrFunction" xml:space="preserve">
     <value>TYPE '{0}' is not declared within a Program or Procedure scope.</value>
   </data>
+  <data name="ExpandingCyclicType" xml:space="preserve">
+    <value>Cannot expand type '{0}' because it is cyclic.</value>
```

Comment:
Is this error message itented for a Cobol user?
If so, the concept of type expansion will not be clear.
The message should rather be something like current error message. See:
https://github.com/TypeCobolTeam/TypeCobol/blob/develop/TypeCobol.Test/Parser/Programs/TypeCobol/CircularReferenceType.rdzMix.txt

Reply:
See ba519a1f4fbf5f087881b8fa0136ca53cd6d1a9f for better messages.